include/anvill/Declarations.h

```diff
@@ -173,6 +157,15 @@ struct CallableDecl {
                       const remill::IntrinsicTable &intrinsics,
                       llvm::BasicBlock *block, llvm::Value *state_ptr,
                       llvm::Value *mem_ptr) const;
+
+  // Try to create a callable decl from a protobuf default callable decl
+  // specification. Returns a string error if something went wrong.
+  //
+  // NOTE(alex): This is following the same pattern as where we decode an entire
```

Comment:
If you're ok with this static member function, we should get rid of this comment.

Reply:
Whoops, I reviewed just as this got merged. I'll follow up with a PR to address these later.